### PR TITLE
P1 lists

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
For the list component. Please review the code and design.

### CODE REVIEW

File: pages/components/list/list.html [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+        <!-- sidebar -->
+        <section class="component-navbar ">
+            <div class="sidebar">
+                <h1 class="component-navbar-heading">Getting Started</h1>
+                <a href="#">
+                    <li class="component-navbar-item">Usage</li>
+                </a>
+
+                <h1 class="component-navbar-heading">Components</h1>
+                <ul>
```
Comment (itsmebipul00) [inline pages/components/list/list.html:75]: ul li a this order

**Review by itsmebipul00 — APPROVED** [review]

**Comment by itsmebipul00** [comment]
Good to merge with changes

File: pages/components/list/list.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
+.list-style-none {
+  list-style: none;
+  padding-inline-start: 0;
+}
+.inline-list {
+  display: flex;
+}
+.list-inline-item:not(:last-of-type) {
+  margin-right: 1.5rem;
+}
```
Comment (bhushannaikaroskar) [inline pages/components/list/list.css:10]: I think these two are connected so maybe you could have used similar naming like for items you would have used inline-list-item or changed list name as list-inline. correct me if I am wrong.

**Review by amartya-rs — COMMENTED** [review]
Looks good to me.

**Review by bhushannaikaroskar — APPROVED** [review]
Rest all I think its good to merge. Just do those small changes if you think I am correct.

File: pages/components/list/list.css (hunk 2 above)
Comment (Omkar-Ghate) [reply pages/components/list/list.css:10]: The first one is for the container, while the second class is for a list item. Hence, made different classes.

**Pull request merged**